src/wp-admin/includes/image.php

```diff
@@ -341,6 +341,18 @@ function wp_create_image_subsizes( $file, $attachment_id ) {
 				 */
 				if ( $scale_down ) {
 					$saved = $editor->save( $editor->generate_filename( 'scaled' ) );
+				} elseif ( $convert ) {
+					/*
+					 * Generate a new file name for the converted image.
+					 *
+					 * As the image file name will be unique due to the changed file extension,
+					 * it does not need a suffix to be unique. However, the generate_filename method
+					 * does not allow for an empty suffix, so the "-converted" suffix is required to
+					 * be added and subsequently removed.
+					 */
+					$converted_file_name = $editor->generate_filename( 'converted' );
+					$converted_file_name = preg_replace( '/-converted\.[a-z0-9]+$/i', '.', $converted_file_name );
+					$saved = $editor->save( $converted_file_name );
```

Comment:
Missing allignment of `=` here.

WPCS fix.

Reply:
I always miss that, thanks. 9057215da8806eacf7f6470e1c1f583f514ee56a